src/nameresolution/builtin.rs

```diff
@@ -70,8 +77,18 @@ pub fn import_prelude<'a>(resolver: &mut NameResolver, cache: &mut ModuleCache<'
 
     // Manually insert some builtins as if they were defined in the prelude
     resolver.current_scope().traits.insert("Int".into(), cache.int_trait);
+
+    // Add string to scope
+    resolver.current_scope().types.insert("string".to_string(), STRING_TYPE);
+    resolver.current_scope().definitions.insert("string".to_string(), BUILTIN_ID);
```

Comment:
Is this what you wanted for the string type?

Reply:
Almost - `BUILTIN_ID` defines the `builtin` symbol which is used in the prelude to define some builtin things like integer addition - so it isn't the write ID for the string constructor. Looks like we don't have a constant for the string constructor id yet - so it'll need to be added like Pair's is.
I also don't think the `string` type itself needs to be imported since it has a keyword that ante knows about (there's a note about this above `define_string`) but as long as its not causing tests to fail it may be fine.

Reply:
Ah I see. I must have mixed it up when reading the lines that defined what builtin was.
I spotted in your z3 branch that you did this work for `string` already. Do you want me to exclude it here?

Reply:
It's probably more appropriate to add it here than my z3 branch. I think I was exploring how easy it'd be to add but forgot to remove it afterward